### PR TITLE
cli: require user to accept privacy policy

### PR DESCRIPTION
**Pull request by triclambert**

### CODE REVIEW

File: packages/cli/src/create.ts [hunk 1]
```diff
@@ -127,16 +127,23 @@ export async function create(): Promise<void> {
...
+        "By selecting 'y' you agree to our terms of service: %s\n\n",
+      url,
+      tos
     );
-    await util.promptForProceed();
   }
 
-  // if (ccfg.cloud_provider === "gcp") {
-  // }
+  if (ccfg.cloud_provider === "gcp") {
```
Comment (triclambert) [inline packages/cli/src/create.ts:140]: I added this here in order to have all messages related to the prompt in a single message to make it as easy-to-read as possible.

File: packages/cli/src/create.ts [hunk 2]
```diff
@@ -125,18 +125,26 @@ export async function create(): Promise<void> {
 }
 
 async function promptForResourceCreation(ccfg: NewRenderedClusterConfigType) {
+  const tos = "https://go.opstrace.com/terms-of-service";
```
Comment (jgehrcke) [inline packages/cli/src/create.ts:128]: `"go/terms-of-service" was not found`

Comment (triclambert) [reply packages/cli/src/create.ts:128]: Yes, I converted to draft until we have the text.

**Comment by triclambert** [comment]
After reconsidering this, I don't believe it is strictly necessary.  Let's abandon it.

**Pull request closed without merging**